contents/blog/hotjar-for-mobile-apps-alternatives.mdx

```diff
@@ -0,0 +1,383 @@
+  <ComparisonRow column1={true} feature="Heatmaps" description="Highlight areas that get the most interaction" />
+  <ComparisonRow column1={true} feature="Scroll maps" description="See how far users scroll and where they dwell" />
+  <ComparisonRow column1={true} feature="Rage taps" description="Identify repetitive taps made out of frustration" />
+  <ComparisonRow column1={true} feature="Dead taps" description="Detect taps on elements that do not respond or trigger an action" />
+  <ComparisonRow column1={false} feature="Crash reports" description="Learn what happens before a crash without complex debugging or reproduction" />
+  <ComparisonRow column1={true} feature="AI summaries" description="Generate summaries of session replays" />
+</ComparisonTable>
+
+#### Other key features
+
+- 📊 **Google Analytics integration:** View your GA data in Clarity and watch recordings for GA segments.
+
+- 🤖 **AI co-pilot:** Chat with an AI to get insights about your recordings.
+
+### How much does Microsoft Clarity cost?
+
+Clarity is completely free.
+
+### Why do companies use Microsoft Clarity?
+
+According to [reviews on G2](https://www.g2.com/products/microsoft-microsoft-clarity/reviews) and [Capterra](https://www.capterra.com/p/236349/Microsoft-Clarity/), it's because:
+
+1. **It's easy to set up and use:** Setting up Clarity doesn't require much technical knowledge and users mostly like the simple user interface.
+
+2. **It helps uncover how users navigate:** Clarity users are mostly using it to understand how people navigate their app and identify opportunities for improvement.
+
+3. **It's free:** The short 30-day retention limit is easy to forgive when it's free.
+
+> #### Bottom line
+> 
+> If you only care about session replay and dont need other features, Clarity is the obvious choice - especially if you're already using Google Analytics. On the other hand, software engineers will find Clarity's lack of debug logs disappointing.
+
+<br />
+
+## 4. PostHog
+
+**Typical users** Engineering and product teams
+  
+![posthog](https://res.cloudinary.com/dmukukwp6/image/upload/v1722527045/posthog.com/contents/posthog-mobile-replay.png)
+
+### What is PostHog?
+
+[PostHog](https://posthog.com) (that's us 👋) is an all-in-one platform built to help engineers create better products. It includes [session replays](/session-replay) **and** a whole bunch more, such as [product analytics](/analytics), [A/B testing](/experiments), and [feature flags](/feature-flags).
+
+It's designed for product-minded engineers, growth teams, and product managers who need to move fast and iterate based on reliable, actionable insights.
+
+#### Mobile replay features
+
+<ComparisonTable column1="PostHog">
+  <ComparisonRow column1="Android, iOS" feature="Supported platforms" description="Mobile platforms where session replay is available" />
```

Comment:
```suggestion
  <ComparisonRow column1="Android, iOS, React Native, Flutter" feature="Supported platforms" description="Mobile platforms where session replay is available" />
```

We should update other alternatives and comparisons that have this data in them while we're here.